src/styles/index.scss

```diff
@@ -1,3 +1,118 @@
-body {
+/* stylelint-disable import-notation */
+@use 'sass:list';
+@import './variables';
```

Comment:
According to the SCSS guidelines, you should prefer `@use` over `@import`. Please, replace the `@import` statement with the `@use` statement.